doc/user/data/sql_funcs.yml

```diff
@@ -129,6 +129,30 @@
   - signature: 'sqrt(x: double precision) -> double precision'
     description: The square root of `x`.
 
+- type: Trigonometric
+  description: Trigonometric functions always take the argument in radians.
```

Comment:
Technically this isn't valid for hyperbolic functions, so we can either remove it, move them to a separate section or add this directly to each one of the functions.

Reply:
I'd say something like "Trigonometric functions take and return `double precision` values representing degrees or radians" and then describe whether degrees or radians within each function!

Reply:
hyperbolic functions don't take degrees either, they take "hyperbolic radians" (not sure if this is a standard term).

https://en.wikipedia.org/wiki/Hyperbolic_angle

Reply:
I renamed this to follow @benesch's suggestion and basically stayed silent about the hyperbolic functions 😬  (the other ones are marked as supporting radians, and I haven't implemented the functions taking degrees right now).

Reply:
It seems that the standard term is just "hyperbolic angle" -- there isn't a unit of measurement (like degrees or radians), probably because there's only one. "Hyperbolic radian" only has a few hundred results on Google, so let's not use that.

We should say something like "circular trigonometric functions take the argument in degrees or radians, and hyperbolic functions take a hyperbolic angle".

I think anyone who is using hyperbolic functions probably already understands what they are and will be able to understand what that means.